src/main/scala/analysis/solvers/FixPointSolver.scala

```diff
@@ -247,8 +249,10 @@ trait PushDownWorklistFixpointSolver[N, T, L <: Lattice[T]] extends MapLatticeSo
   def process(n: N): Unit =
     val xn = x(n)
     val y = transfer(n, xn)
+    if (y != xn) {
+      for succ <- outdep(n) do propagate(y, succ)
+    }
```

Comment:
@yousifpatti didn't you find that doing this caused some bugs?

Reply:
This caused the results of constantprop analysis to not be propagated properly. The effects may not be noticeable because evaluate expression may return TOP and you deal with it the same way. The best way to check is by checking the dot file of constant prop to see if values are being propagated.

Reply:
Thanks, I remember that made the analyses resolve some indirect calls incorrectly. It would be good if we could figure out how to make this optimisation work but it should be reverted for now.